tests/package.yaml

```diff
@@ -17,6 +17,9 @@ dependencies:
   host:
     url: github.com/toitlang/pkg-host
     version: ^1.15.1
+  partition-table:
```

Comment:
Do we need this?

Reply:
yes. We are including the `firmware` library, which needs the partition-table package.